plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/AbstractTestOracleTypeMapping.java

```diff
@@ -106,7 +106,7 @@ public void setUp()
 
     private DataSetup trinoCreateAsSelect(String tableNamePrefix)
     {
-        return new CreateAsSelectDataSetup(new TrinoSqlExecutor(getQueryRunner()), tableNamePrefix);
```

Comment:
Commit message could be a one-line "Reuse trinoCreateAndInsert and trinoCreateAsSelect in type mapping tests".